prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -142,7 +141,6 @@ class WorkflowEngine(
                 retryJson,
                 txn
             )
-            recordAction(actionHistory, txn)
         }
```

Comment:
A nice side effect of changing actionHistory in SendFunction is that it was no longer needed down here.